src/components/answersList/styles.css

```diff
@@ -0,0 +1,36 @@
+.answers-list-table {
```

Comment:
Adding `border-spacing: 0` will prevent the table header from moving slightly up and down while scrolling.

Reply:
I seem to have been wrong and the shaking of the header still persists. Removing the padding from `th`s and `td`s makes it less prominent but does not solve the problem. Not sure what exactly causes it.